client/ts/package.json

```diff
@@ -15,6 +12,17 @@
     "telemetry",
     "control systems"
   ],
+  "homepage": "https://github.com/synnaxlabs/synnax/tree/main/client/ts",
+  "bugs": {
+    "url": "https://github.com/synnaxlabs/synnax/issues"
+  },
+  "license": "BUSL-1.1",
```

Comment:
Just `BSL` not `BUSL`

Reply:
It is `BUSL-1.1` in the [SPDX license](https://spdx.org/licenses/BUSL-1.1.html) database. [BSL](https://spdx.org/licenses/BSL-1.0.html) is the Boost Software License.